package_build_list_host_windows.json

```diff
@@ -5,6 +5,7 @@
     "comment4" : "Note:  Build from source occurs before build_from_folder",
     "build_from_source": {
         "assimp-5.1.6-rev1-windows":  "Scripts/extras/pull_and_build_from_git.py ../../package-system/assimp --platform-name Windows --package-root ../../package-system --clean",
+        "AWSGameLiftServerSDK-3.4.2-rev1-windows": "package-system/AWSGameLiftServerSDK/build_package_image.py --platform-name windows",
```

Comment:
Same question as above: Seems like GAMELIFT_SERVER_SDK_RELEASE_VERSION is 4.0.2. Then why are we specifying 3.4.2 here?